src/cigar.py

```diff
@@ -65,4 +73,18 @@ def edits_to_cigar(edits: str) -> str:
     '1M1D6M1I4M'
 
     """
-    return ""
+
+    cigar = ''
+    blocks = split_blocks(edits)
+
+    for b in blocks:
+        cigar + str(len(b)) + b[0]
```

Comment:
This is not doing anything. You are creating a new string by concatenating `cigar`, `str(len(b))` and `b[0]`, but you do not store it anywhere. I think you meant `cigar += str(len(b)) + b[0]`.